js/hollaex.js

```diff
@@ -189,30 +189,45 @@ module.exports = class hollaex extends Exchange {
         for (let i = 0; i < keys.length; i++) {
             const key = keys[i];
             const market = pairs[key];
-            const id = this.safeString (market, 'name');
             const baseId = this.safeString (market, 'pair_base');
             const quoteId = this.safeString (market, 'pair_2');
             const base = this.commonCurrencyCode (baseId.toUpperCase ());
             const quote = this.commonCurrencyCode (quoteId.toUpperCase ());
-            const symbol = base + '/' + quote;
-            const active = this.safeValue (market, 'active');
-            const maker = this.fees['trading']['maker'];
-            const taker = this.fees['trading']['taker'];
             result.push ({
-                'id': id,
-                'symbol': symbol,
+                'id': this.safeString (market, 'name'),
+                'symbol': base + '/' + quote,
                 'base': base,
                 'quote': quote,
+                'settle': undefined,
                 'baseId': baseId,
                 'quoteId': quoteId,
+                'settleId': undefined,
                 'type': 'spot',
                 'spot': true,
-                'active': active,
+                'margin': false,
+                'swap': false,
+                'future': false,
+                'option': false,
+                'active': this.safeValue (market, 'active'),
+                'contract': false,
+                'linear': undefined,
+                'inverse': undefined,
+                'taker': this.fees['trading']['maker'],
+                'maker': this.fees['trading']['taker'],
```

Comment:
Do we want to start adding these fields in to create a more unified structure?

Reply:
Yes, we do, but we don't want those properties to be hardcoded, if possible.

Reply:
The markets are extended with `fees` in the base method `setMarkets`, so we do't have to add them here.

Reply:
> The markets are extended with `fees` in the base method `setMarkets`, so we do't have to add them here.

Unless I code these  keys in, the output of fetchMarkets doesn't include it, look at `digifinex` for example

Reply:
@samgermain fetchMarkets doesn't, but loadMarkets does. It is done in the base class in the `setMarkets` helper.